openfisca_aotearoa/variables/acts/social_security/accommodation_supplement/accommodation_supplement.py

```diff
@@ -72,12 +80,406 @@ class accommodation_supplement__below_income_threshold(Variable):
+        whose accommodation costs are rent or payments for board and lodgings
+        """
+    situation_2 = """
+        To a person who has no dependent children and who is in a relationship,
+        or a sole parent with 1 dependent child, whose accommodation costs are
+        rent or payments for board and lodgings
+        """
+    situation_3 = """
+        To any other person whose accommodation costs are rent or payments for
+        board and lodgings
+        """
+    situation_4 = """
+        To a person who has 1 or more dependent children and who is in a
+        relationship, or a sole parent with 2 or more dependent children, whose
+        accommodation costs are the sum of payments required under any mortgage
+        security, and other payments that the chief executive is satisfied are
+        reasonably required to be made in respect of the person’s home
+        """
+    situation_5 = """
+        To a person who has no dependent children and who is in a relationship,
+        or a sole parent with 1 dependent child, whose accommodation costs are
+        the sum of payments required under any mortgage security, and other
+        payments that the chief executive is satisfied are reasonably required
+        to be made in respect of the person’s home
+        """
+    situation_6 = """
+        To any other person whose accommodation costs are the sum of payments
+        required under any mortgage security, and other payments that the chief
+        executive is satisfied are reasonably required to be made in respect of
+        the person’s home
+        """
+
+
+class accommodation_supplement__situation(Variable):
+    label = "TODO"
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/whole.html#DLM6784877"
+    documentation = """TODO"""
+    entity = Person
+    value_type = Enum
+    possible_values = AccommodationSupplement__Situation
+    default_value = AccommodationSupplement__Situation.unknown
+    definition_period = DateUnit.WEEK
+
+    def formula_2018_11_26(people, period, _params):
+        families = people.family
+        last_week = period.last_week
+        family_members = families.members("social_security__dependent_child", last_week)
+        dependent_children = sum(family_members)
+        partners = families.nb_persons(Family.PARTNER)
+
+        ssa_sched_4_part_7_1 = (
+            + (
+                + (dependent_children >= 1) * (partners >= 1)
+                + (dependent_children >= 2) * (partners == 0)
+                )
+            * AccommodationSupplement__Situation.situation_1.index
+            )
+
+        ssa_sched_4_part_7_2 = (
+            + (
+                + (dependent_children == 0) * (partners >= 1)
+                + (dependent_children == 1) * (partners == 0)
+                )
+            * AccommodationSupplement__Situation.situation_2.index
+            )
+
+        ssa_sched_4_part_7_3 = False  # TODO: Add "for board and rent"
```

Comment:
§ 65

Reply:
Accommodation type: 

1. _renting_, _boarding_or_lodging_ (consider 62% of it), 
2. _mortgage_, 
3. _social_housing_ (ineligibility if), _residential_care_.

Accommodation amount: weekly.
Accommodation costs: weekly + type.

Reply:
Regarding residential care, we won't consider it as for now.